README.md

```diff
@@ -108,12 +93,14 @@ When archiving your application for submission to the App Store or TestFlight, X
 In the `ViewControler.swift` file, update the variables at the top of this file with the information for your tenant.
 
 ```swift
-    let kTenantName = "<tenant>.onmicrosoft.com" // Your tenant name
-    let kClientID = "<your-client-id>" // Your client ID from the portal when you created your application
-    let kSignupOrSigninPolicy = "<your-signin-policy>" // Your signup and sign-in policy you created in the portal
-    let kEditProfilePolicy = "<your-edit-profile-policy>" // Your edit policy you created in the portal
-    let kGraphURI = "<Your backend API>" // This is your backend API that you've configured to accept your app's tokens
-    let kScopes: [String] = ["<Your backend API>/demo.read"] // This is a scope that you've configured your backend API to look for.
+    let kTenantName = "fabrikamb2c.onmicrosoft.com" // Your tenant name
+    let kAuthorityHostName = "fabrikamb2c.b2clogin.com" // Your authority host name

```

Comment:
When I click Authorize, it seems to open localhost instead of SUSI(login) page . Can you confirm?

Reply:
Reply URL was changed back to our client id in portal, should be alright now.